src/renderer/scss/init/_gui.scss

```diff
@@ -0,0 +1,204 @@
+// Generic html styles used accross the App
```

Comment:
I'm not able to highlight anything in the app. We shouldn't be preventing this.

Reply:
Do we need the ability to highlight everything? Titles, markdown content, wallet addresses and so on make sense but not things like transaction fees.

Reply:
Probably not everything, but I don't think we should prevent highlighting if users want to. It's a lot more work to have to opt into something if we want it to be highlightable.